packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -1201,6 +1201,12 @@ export class LitNodeClientNodeJs
       responseData
     ) as NodeShare;
 
+    const responseFromStrategy: any = processLitActionResponseStrategy(
+      responseData,
+      params.responseStrategy ?? { strategy: 'leastCommon' }
```

Comment:
I don't see `responseStrategy` as a valid option for `JsonExecutionSdkParams`, yet we check for it in the new code.  If the idea is that `executeJs()` needs to support `responseStrategy`, shouldn't the `JsonExecutionSdkParams` be updated also?

Reply:
Missed it in the new def :+1: